os-eap64-launch/added/launch/datasource.sh

```diff
@@ -26,3 +26,8 @@ function inject_datasources() {
 function generate_datasource() {
   generate_datasource_common "${1}" "${2}" "${3}" "${4}" "${5}" "${6}" "${7}" "${8}" "${9}" "${10}" "${11}" "${12}" "${13}" "${14}"
 }
+
+function refresh_interval() {
```

Comment:
Would it be better to stub this out in datasource-common.sh, then have it be overridden by the datasource.sh file that is including it?  My concern is that this is a required method, but may not be available in other scripts that utilize it (i.e. we may break existing users who are not aware of this change).  A simple test would be to move this implementation into datasource-common.sh and use it as the default.